### ISSUE TITLE
[vizbuilder] "margin of error" checkbox toggle

### ISSUE THREAD

**Issue by davelandry** [Datawheel/canon#455]
- [x] use the blueprint `Checkbox` component
- [x] label it "Show Margin of Error"
- [x] position it underneath the measure selector, underneath the measure description (if it has one)
- [ ] only display the checkbox if the selected measure has error measures associated with it
- [x] make it "off" by default
- [ ] when "off", do not load/display the margins of errors in the visualizations (not loading them will help increase speed)

**Comment by davelandry** [comment]
@frabarz it looks like there is 1 missing feature here: if the checkbox of "off", we do not want to even load the margins of error from the database. This is because some margin of errors take some time to calculate, so having this checkbox turned off will be faster to load.

**Comment by frabarz** [comment]
I just found a kind of loop in the process: in issue #447 we disabled margins of error if there are more than 12 members in the dataset. If I try to enable MoE and the result of the fetch contains more than 12 members, the confidence intervals are loaded but disabled anyways, and is confusing to the user expecting to see them. Should I...
- remove the 12 members check?  
this gives the user the option to add the CI, but can lead to charts [like this one](https://user-images.githubusercontent.com/11713191/53757347-2e0cfe80-3e9a-11e9-844a-3f2d5e14b0cd.png) again
- hide the checkbox if there's more than 12 members?  
after first query, which is always without MoE, I can know how many members we are handling in the charts, so I can hide the checkbox. However this would mean making less info available to the user, at least until they cut to fewer members
